drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -1154,47 +1154,12 @@ static int bcm2835_isp_get_supported_fmts(struct bcm2835_isp_node *node)
 		const struct bcm2835_isp_fmt *fmt = get_fmt(fourccs[i]);
 
 		if (fmt) {
-			list[j] = *fmt;
+			list[j] = fmt;
```

Comment:
Is this change covered by the commit message? Doesn't it belong more with the other commit?

Reply:
:-(
git add -i split it not quite where I thought. Yes it's part of the other commit.